core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -531,6 +531,23 @@ public State state() {
     return mState.get();
   }
 
+  @Override
+  public boolean append(PageId pageId, int appendAt, byte[] page, CacheContext cacheContext) {
+    LOG.debug("append ({},{} bytes) enters", pageId, page.length);
```

Comment:
Not sure whether these debug logs are useful at all. But if you add one for "enters" maybe also add one for "exists". Though I'd rather vote for removing them all.

Reply:
removed the debug logs